opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/mqtt/OpensyncMqttClient.java

```diff
@@ -171,24 +189,22 @@ public void run() {
                         // new Topic("#", QoS.AT_LEAST_ONCE),
                         // new Topic("test/#", QoS.EXACTLY_ONCE),
                         // new Topic("foo/+/bar", QoS.AT_LEAST_ONCE)
-                        Topic[] topics = { new Topic("#", QoS.AT_LEAST_ONCE), };
+                        Topic[] topics = {new Topic("#", QoS.AT_LEAST_ONCE),};
```

Comment:
Why keep the trailing comma? This is more a format issue than a code issue, I'm just wondering!

Reply:
It is perfectly acceptable for arrays, enums, etc in java to have a hanging comma, some feel it is easier to read diffs etc.
In this case, it is a single line, and no other topics have been added. I actually didn't change it, the formatter did, which is why it shows up as a delta here. However, for reference: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8